src/uniprotkb/__mocks__/entryModelData.json

```diff
@@ -1350,5 +1350,44 @@
       "rank": "family",
       "hidden": true
     }
-  ]
+  ],
```

Comment:
Not really willing to change that one in this specific PR. It's used in 21 test files and TypeScript is complaining about invalid schema in plenty of them about other parts of this object.
Sounds like it will open a can of worms

Reply:
Yeah ok, sounds like it would deserve its own task.